Formula/unbound.rb

```diff
@@ -13,11 +13,13 @@ class Unbound < Formula
 
   depends_on "openssl"
   depends_on "libevent"
+  depends_on "expat"
```

Comment:
Strangely on my machine I could not build without including the expart
module from homebrew

$ uname -a

Darwin ... 15.5.0 Darwin Kernel Version 15.5.0: Tue Apr 19 18:36:36 PDT 2016

On Mon, Jul 18, 2016 at 1:21 PM Dominyk Tiller <notifications@github.com>
wrote:

> In Formula/unbound.rb
> <https://github.com/Homebrew/homebrew-core/pull/3080#discussion_r71179279>
> :
>
> > @@ -13,11 +13,13 @@ class Unbound < Formula
> >
> >    depends_on "openssl"
> >    depends_on "libevent"
> > +  depends_on "expat"
>
> Does it need a newer version than the system provided one now?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Homebrew/homebrew-core/pull/3080/files/bae0c807e72cb3fde2dff0a9fabb6c40486b6982#r71179279>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATkHVatgdE8EE1hjYXhcRDsJbK87oDeUks5qW6gAgaJpZM4JOknY>
> .
>